03-workshop_import_data.Rmd

```diff
@@ -156,23 +156,23 @@ Where just one line copied:
 ]
 
 
----
+---  
```

Comment:
The 2 spaces after --- causes an issue with the slides as they now run into each other.